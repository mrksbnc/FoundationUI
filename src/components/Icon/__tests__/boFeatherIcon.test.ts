import {
	BoFeatherIcon,
	Icon,
	IconSize,
	IconStrokeStyle,
} from '@/components/Icon';
import { mount } from '@vue/test-utils';
import { describe, expect, it } from 'vitest';

describe('BoIcon', () => {
	it('renders properly', () => {
		const wrapper = mount(BoFeatherIcon, {
			props: {
				name: Icon.activity,
				size: IconSize.md,
				strokeStyle: IconStrokeStyle.normal,
			},
		});
		expect(wrapper.html()).toMatchSnapshot();
	});
});
