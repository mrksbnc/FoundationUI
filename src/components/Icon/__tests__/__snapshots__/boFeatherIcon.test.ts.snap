// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BoIcon > renders properly 1`] = `
"<!-- NOTE: These icons are part of the bo-feather-icon icon set. -->
<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke-width="2" stroke="var(--icon-color)" stroke-linecap="round" stroke-linejoin="round" class="bo-feather-icon bo-feather-icon---activity">
  <polyline points="22 12 18 12 15 21 9 3 6 12 2 12"></polyline>
</svg>"
`;
